### DISCUSSION TITLE
Issue with types [ltijs]

### DISCUSSION THREAD

**Discussion by Rafa97485** [DefinitelyTyped/DefinitelyTyped#66880] [Issues with a @types package]
in the file Provider.d.ts the method deletePlatform should have a second parameter clientId (line 89)
like this: deletePlatform(url: string, clientId: string): Promise<boolean>;

[a comment by an automated account is not shown]

**Comment by paulschwoerer** [comment]
You're absolutely right, thanks for pointing it out. Would you like to open a PR fixing the issue?

**Reply by Rafa97485** [reply]
I tried to push my branch and I don't have permission

**Reply by paulschwoerer** [reply]
See [this tutorial](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) on how to contribute to projects. You probably need to fork the DefinitelyTyped repository and then open a PR.

**Comment by Semigradsky** [comment]
Related: https://github.com/Cvmcosta/ltijs/issues/197

**Reply by paulschwoerer** [reply]
Would be great to have the types integrated into ltijs directly.
